templates/user/edit.html.twig

```diff
@@ -15,7 +15,7 @@
     {{ include('user/_delete_form.html.twig') }}
 
     {% for role in user.roles %}
-        {% if role == 'ROLE_ADMIN' %}
+        {% if role == 'ROLE_COMM' %}
```

Comment:
Is it not better to do :
`{% if role == 'ROLE_COMM' or role == 'ROLE_ADMIN' %}`

Reply:
you are right